src/collections/_documentation/development/server/environment.md

```diff
@@ -1,57 +1,38 @@
 ---
 title: 'Development Environment'
-sidebar_order: 1
+sidebar_order: 2
 ---
 
-## Setting up Your Development Environment
+This guide steps you through configuring a local development environment for the Sentry server on macOS. If you're using another operating system the instructions are still roughly the same, but we don't maintain any official documention.
 
-### Linux
+#### A quick note on Bash vs Zsh...
 
-Follow [the Docker file in the repository](https://github.com/getsentry/sentry/blob/master/docker/Dockerfile) but instead of creating and installing a wheel, install the package in editable mode (`pip install . -e`).
+macOS Catalina will recommend that you use `zsh` instead of `bash`. While the differences are minimal, this guide primarily covers Bash. Verify your shell by running `echo $SHELL`.
```

Comment:
Agreed - I actually dont think we really need this callout - was just preserving it. Only a few places it actually applies (e.g. Volta) and if we just say "use the official docs to install those things" we're probably in a good spot.

Reply:
Aside the reason its _not_ a note is because this pagee was filled with those kinds of callout boxes and it's extremely distracting visually. I tried to remove the alert boxes as most things are just instructions. I kid you not the page was like 20% visually alert boxes (and still has quite a few)

Reply:
That works for me as long as you remove this section completely or at least remove the ellipsis from the title 😀